airflow/providers/amazon/aws/sensors/eks.py

```diff
@@ -98,13 +99,15 @@ def __init__(
         self.region = region
         super().__init__(**kwargs)
 
-    def poke(self, context: Context):
-        eks_hook = EksHook(
+    @cached_property
+    def hook(self):
+        return EksHook(
             aws_conn_id=self.aws_conn_id,
             region_name=self.region,
         )
```

Comment:
I wonder if it makes sense to put the cached hook in a parent BaseEksSensor that these inherit?   Likely overkill.....

Reply:
If there was already a base sensor, yeah, sure. To add it just for this, idk...

Reply:
That's fair.  :+1:   Feel free to resolve/close this.

Reply:
looking a bit closer at it, I think there would be more use cases for a base sensor here, but I'll do that in a different PR.